### PR TITLE
ukrainian localization

### PR DESCRIPTION
**Pull request by web-ruslan**
ukrainian localization fix

### CODE REVIEW

**Review by web-ruslan — COMMENTED** [review]
Review needed @zaaom @fletch3555 @marktopper 
Thank You for help!

File: publishable/lang/uk/database.php [hunk 1]
```diff
@@ -1,55 +1,32 @@
...
     'delete_bread_before_table' => 'Перед видаленням таблиці обов\'язково видаліть BREAD таблиці.',
-    'delete_table_bread_conf'   => 'Так, видалити BREAD',
-    'delete_table_bread_quest'  => 'Ви впевнені, що хочете видалити BREAD для таблиці :table?',
     'delete_table_confirm'      => 'Так, видалити таблицю',
     'delete_table_question'     => 'Ви дійсно хочете видалити таблицю :table?',
-    'description'               => 'Опис',
-    'display_name'              => 'Відображена назва',
-    'display_name_plural'       => 'Відображена назва (в множині)',
-    'display_name_singular'     => 'Відображена назва (в однині)',
-    'edit_bread'                => 'Редагувати BREAD',
```
Comment (zaaom) [inline publishable/lang/uk/database.php:32]: have been away from the repo for a while. but why these keys are removed?

Comment (web-ruslan) [reply publishable/lang/uk/database.php:32]: @zaaom these keys are removed from english version https://github.com/the-control-group/voyager/blob/1.6/publishable/lang/en/database.php

**Review by zaaom — APPROVED** [review]

**Comment by web-ruslan** [comment]
@zaaom still review is required :(

**Comment by web-ruslan** [comment]
Review needed @zaaom  @fletch3555  @marktopper 
Thank You for help!

**Comment by zaaom** [comment]
@web-ruslan you need a review from a maintainer to merge.

**Comment by web-ruslan** [comment]
Review needed @emptynick 
Thank You for help!

**Review by zaaom — APPROVED** [review]

**Comment by web-ruslan** [comment]
@tnylea please check this code and merge it

**Pull request closed without merging**